jvm-packages/xgboost4j-spark/src/main/scala/ml/dmlc/xgboost4j/scala/spark/XGBoost.scala

```diff
@@ -315,8 +314,17 @@ object XGBoost extends Serializable {
       case _ => throw new IllegalArgumentException("parameter \"tracker_conf\" must be an " +
           "instance of TrackerConf.")
     }
+    val timeoutRequestWorkers: Long = params.get("timeout_request_workers") match {
+      case None => 30 * 60 * 1000L
```

Comment:
None means the user does not mind waiting forever? shouldn't this be 0?

Reply:
Done